packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -34,7 +34,7 @@ class CheckboxGroup extends React.Component {
             name={option.name}
             value={option.value}
             checked={includes(this.props.values, option.value)}
-            onChange={val => this.handleChange(option.value, val)}/>;
+            onChange={e => this.handleChange(option.value, e.target.checked)}/>;
```

Comment:
> We'd like to preserve the ability to use refs if we need to.

I mean we would use always both. You still need class components in case of lifecycle hooks (`componentDidUpdate()`) but if you can (vast majority of cases), it's nicer to use functional components.